Make sure install dir exists before copying file to it.

--- a/LablGlut/src/Makefile
+++ b/LablGlut/src/Makefile
@@ -68,6 +68,7 @@
 	cd "$(INSTALLDIR)" && $(RANLIB) liblablglut$(XA)
 	@if test -f dlllablglut$(XS); then $(MAKE) installdll; \
 	 else $(MAKE) installtop; fi
+	mkdir -p "$(BINDIR)"
 	cp lablglut$(XB) "$(BINDIR)"
 
 real-install: preinstall
